index.html

```diff
@@ -96,15 +96,14 @@ <h4>Skills</h4>
                     <a href="#"><img alt= 'twitter' src="./assets/Twitter.svg"></a>
                     <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
                 </div> 
-                
                     <span class="bottom-span">&#8595; &nbsp;&nbsp; WORK</span>
             </div>   
         </section>
         <section class="work-flex" id="work-flex">
             <div class="caption">
-                <h1>MY RECENT 
+                <h2>MY RECENT 
```

Comment:
- [OPTIONAL ] I think it would be great if you moved your works section to generate dynamically with js file since it is one of the requirements 

![js](https://user-images.githubusercontent.com/58553711/181749215-5f794614-b96e-4676-9671-1fac3e42eda2.PNG)

Reply:
@Reem-lab I will definitely act upon your recommendations.